docs/releasing.md

```diff
@@ -0,0 +1,61 @@
+# Overview
+
+This document describes the release flow and how to create a bundle release.
+
+## Latest tag 
+
+Every change in the master branch which passes all checks triggers creating a `latest` tag and a [latest release](https://github.com/kyma-project/bundles/releases/tag/latest).
```

Comment:
The `latest relase` link doesn't work. Is it expected behavior, source is not created yet?

Reply:
I've removed links